preload_data.sh

```diff
@@ -57,7 +57,7 @@ function main() {
     copy_resource "configmap" "ibm-cpp-config"
     copy_resource "configmap" "common-web-ui-config"
     copy_resource "configmap" "platform-auth-idp"
-    copy_resource "commonservice" "common-service"
+    copy_resource "commonservice" "common-service" "common-service-from-$FROM_NAMESPACE"
```

Comment:
```suggestion
    copy_resource "commonservice" "common-service" "preload-common-service-from-$FROM_NAMESPACE"
```

Suggest to leave a more specific `preload` identifier for easier filtering in the future, and also helps L2 support know where resource came from.